src/main/java/xyz/nucleoid/plasmid/command/GameChannelCommand.java

```diff
@@ -37,7 +38,7 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
         dispatcher.register(
             literal("game")
                 .then(literal("channel")
-                    .requires(source -> source.hasPermissionLevel(3))
+                        .requires(Permissions.require("plasmid.command.game_channel"))
```

Comment:
Change to Permissions.require("plasmid.command.game_channel", 3) which would make 3 a default in case the server isn't using a permission mamanger.